dev-env/run-env.sh

```diff
@@ -6,7 +6,7 @@ export DATAVERSE_IMAGE_TAG=$1
 export COMPOSE_HTTP_TIMEOUT=200
 
 # Timeout for Dataverse bootstrap configbaker
-export DATAVERSE_BOOTSTRAP_TIMEOUT="5m"
+export DATAVERSE_BOOTSTRAP_TIMEOUT="10m"
```

Comment:
Let's keep it at 5m. It is a value that has proven to be enough for GitHub actions.

In fact, I think this value should be set in a .env file. That way, the environment runner sets the value that best suits (Potentially higher values for local environments and shorter values for actions).

We don't need to change that now. For this PR, I think it is enough to change it back to 5m.